less/_grid.less

```diff
@@ -181,6 +181,37 @@
 				padding-bottom: 0;
 			}
 		}
+	}
+
+	// 3 COLUMNS
+	[class*='col'] {
+		@media @mq-md {
+			.sss&,
+			.ssw& {
```

Comment:
Yes, yes, the `.sss` and `.ssw` classes are the same, visually, but if we had three columns of the same width (`.col3`) then using the `.ssw` class wouldn't make sense.